packages/react-instantsearch/examples/geo-search/README.md

```diff
@@ -3,7 +3,7 @@ This example shows how to perform a geo search using `react-instantsearch`.
 To start the example:
 
 ```sh
-yarn install --no-lockfile
+yarn install
```

Comment:
It's there to not create a lock file when checkouting and running the examples. (and its present in every examples).

No matter what, this is not related to the dependencies upgrade.